pyscript.core/src/stdlib/pyweb/media.py

```diff
@@ -0,0 +1,91 @@
+from pyodide.ffi import to_js
+from pyscript import window
+
+
+class Device:
+    def __init__(self, device):
+        self._js = device
+
+    @property
+    def id(self):
+        return self._js.deviceId
+
+    @property
+    def group(self):
+        return self._js.groupId
+
+    @property
+    def kind(self):
+        return self._js.kind
+
+    @property
+    def label(self):
+        return self._js.label
+
+    def __getitem__(self, key):
+        return getattr(self, key)
+
+    @classmethod
+    async def load(cls, audio=False, video=True):
+        options = window.Object.new()
+        options.audio = audio
+        if isinstance(video, bool):
+            options.video = video
+        else:
+            # TODO: Think this can be simplified but need to check it on the pyodide side
+
+            # TODO: this is pyodide specific. shouldn't be!
+            options.video = window.Object.new()
+            for k in video:
+                setattr(
+                    options.video,
+                    k,
+                    to_js(video[k], dict_converter=window.Object.fromEntries),
+                )
+
+        stream = await window.navigator.mediaDevices.getUserMedia(options)
+        return stream
+
+    async def get_stream(self):
+        key = self.kind.replace("input", "").replace("output", "")
+        options = {key: {"deviceId": {"exact": self.id}}}
+
+        print("optioniiiiii", options)
```

Comment:
is this meant to be kept in the code? if yes, could we make it a `window.console.debug(...)` instead ? we already had users complaining we log too much and at least via the explicit debug method, unless explicitly enabled, these logs would not show up.

Reply:
Man... I hate when I miss those debug statements 🤦 . This should go